scripts/dependencycache.ps1

```diff
@@ -0,0 +1,26 @@
+Function Step-Main {
+    Param (
+		[string]$Command = "default",
+		[string]$Manager = "yarn"
+    )
+
+    Get-Process {
+        switch ( $Command ) {
+            clear {
+                Remove-Item -Recurse -Force dist
+				Remove-Item -Recurse -Force node_modules
+				switch ($Manager) {
+					yarn {
+						Remove-Item -Recurse -Force $(yarn cache dir)
+					}
```

Comment:
```suggestion
						Remove-Item -Force -ErrorAction Ignore yarn.lock
						yarn install
					}
```

Also remove the lockfile which holds the Git SHA then run yarn install

Reply:
That'd force the update and end up updating some hashes unintentionally. `yarn upgrade-interactive --latest` should be used instead.